client/components/Tracker/Form/Form.jsx

```diff
@@ -1,11 +1,21 @@
 import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import styled from 'styled-components';
 
 const Form = () => {
   const [feeling, updateFeeling] = useState('');
   const [thoughts, updateThoughts] = useState('');
 
   const handleSubmit = (event) => {
+    //(user_id, thought, feeling_id, track_date)
+
+    const trackDate = new Date(Date.now()).toLocaleString().split(',')[0].split('/').join('-');
+
+    axios.post('/tracker/postListItem', {
+      feeling, thoughts, trackDate,
+    })
+      .then(() => console.log('succesful post to the DB'))
```

Comment:
To consider later - rendering out the latest post if successful?

Reply:
Sounds good! going to implement in a separate branch!